pkg/compiler/debug.go

```diff
@@ -201,6 +201,21 @@ func (c *codegen) scTypeFromExpr(typ ast.Expr) string {
 	if c.typeOf(typ) == nil {
 		return "Any"
 	}
+	if named, ok := t.(*types.Named); ok {
+		if isInteropPath(named.String()) {
+			name := named.Obj().Name()
+			pkg := named.Obj().Pkg().Name()
+			switch pkg {
+			case "blockchain", "contract":
+				return "Array" // Block, Transaction, Contract
```

Comment:
But they're `Struct` in neo-go, even though syscalls return `Array`.

Reply:
I think, interop methods signature must be altered to return pointers, after it was made possible.

Reply:
Yeah, that would be ideal, avoiding conversions and matching this interface with other languages (suppose you want to expose some method that accepts a `Transaction`, C# code would pass an `Array` for it while we would be expecting a `Struct`).